package.json

```diff
@@ -2,8 +2,10 @@
   "name": "@tl-its-umich-edu/react-ga-onetrust-consent",
   "version": "1.0.0",
```

Comment:
Should the package version be updated to new changes like 1.0.1? Looks like when you do the `npm install` the package is built as ` @tl-its-umich-edu/react-ga-onetrust-consent@1.0.0` . 

But could it be also be controlled with Github Action build/release?

Reply:
I agree this should be an iteration change, I'll look into how to add it to github actions